tests/conftest.py

```diff
@@ -71,7 +71,7 @@ def supported_configuration():
 
     node_obj = ocp.OCP(kind=constants.NODE)
     log.debug('Checking if system meets minimal requirements')
-    nodes = node_obj.get().get('items')
+    nodes = node_obj.get(selector=constants.WORKER_LABEL).get('items')
```

Comment:
In general, I don't think that we can assume that every worker node is a storage node, but as long as our CI env is concerned, this probably holds.